### PR TITLE
Update kapp-controller version doc. #1491

### LINKED ISSUE #1491: Provide documented workaround for changing `kapp-controller` version
## Feature Request

Internally (at VMware), a request to swap out `kapp-controller` for a different version was mentioned.

Today, we do not have this ability until we have this ability until we solve for https://github.com/vmware-tanzu/tanzu-framework/issues/60 as kapp-controller is referenced in the bom.

For a workaround, we need someone to document the steps to:

**management-clusters**

* pause the reconciled (workload) kapp-controller from the management cluster
* re-deploy an updated kapp-controller in the workload cluster

**standalone clusters**

* swap the kapp-controller

## Describe alternatives you've considered

None

## Additional context

None

### PR DESCRIPTION
**Pull request by jorgemoralespou**
## What this PR does / why we need it
Provides a document with the process to update kapp-controller on both managed clusters and standalone clusters

## Details for the Release Notes (PLEASE PROVIDE)
```release-note
doc on updating kapp-controller version
```

## Which issue(s) this PR fixes
Fixes: #1491

## Describe testing done for PR
make mdlint
hugo server

## Special notes for your reviewer
Formating might not be optimal or inline with other pages.

Signed-off-by: Jorge Morales Pou jomorales@vmware.com

### CODE REVIEW

**Comment by kcoriordan** [comment]
Is there meant to be a `upgrade-kapp-controller` in this PR?

**Comment by jorgemoralespou** [comment]
@kcoriordan somehow was not added to push.

File: docs/site/content/docs/latest/upgrade-kapp-controller.md [hunk 1]
```diff
@@ -0,0 +1,182 @@
+# Upgrade kapp-controller
+
+Tanzu Community Edition provides a built in version of kapp-controller that is dictated
+by the Bill of Materials (BOM) of the Tanzu Kubernetes Release (TKR) in use.
+
+It is possible to upgrade kapp-controller version provided. Depending on your type of
```
Comment (kcoriordan) [inline docs/site/content/docs/latest/upgrade-kapp-controller.md:6]: change to
It is possible to upgrade **the** kapp-controller version provided

File: docs/site/content/docs/latest/upgrade-kapp-controller.md [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+cluster you will need to follow one of the available approaches.
```
Comment (kcoriordan) [inline docs/site/content/docs/latest/upgrade-kapp-controller.md:7]: might be clearer to say:
Depending on whether you are running a standalone or management cluster, you will need to follow one of the following approaches.

**Review by kcoriordan — DISMISSED** [review]

**Review by stmcginnis — APPROVED** [review]
Kay's comments have been addressed, and this looks good to me.

**Pull request merged**